airbyte-oauth/src/main/java/io/airbyte/oauth/flows/SurveymonkeyOAuthFlow.java

```diff
@@ -56,7 +77,15 @@ protected String formatConsentUrl(final UUID definitionId,
 
   @Override
   protected String getAccessTokenUrl(final JsonNode inputOAuthConfiguration) {
```

Comment:
is there a reason not to reuse the logic in https://github.com/airbytehq/airbyte/pull/17964/files#diff-5c30eabeb5bb52b1031725d5bc944852ffbfaa1b71491aa249978109a38eee64R48 ?